pages/admin/request/create-replacement.tsx

```diff
@@ -219,7 +221,7 @@ export default function CreateReplacement() {
                 type: 'REPLACEMENT',
                 receiveEmailUpdates: false,
               },
-              paymentInformation: INITIAL_PAYMENT_DETAILS,
+              paymentInformation: ZERO_REPLACEMENT_FEE_PAYMENT_DETAILS,
```

Comment:
Is it $0 replacement fee for all reasons or only for mail lost?

Reply:
I would assume only for mail lost (since that would not be the fault of the client) but something like "Lost" probably would be. The RCD request though isn't super clear, and we could clarify? What do you think?

The request from RCD was as follows "how a default permit fee of $0 without any specified payment method."

Reply:
As discussed in meeting, I think current implementation sets $0 for all reasons, can you please double check? :)

Reply:
Yes it is. Now with the most recent changes the default replacement fee is changed back to $31 and becomes $0 if Mail Lost is selected as the reason for replacement. It stays goes back to $31 if either Lost or Stolen is selected as a reason. What do you think? 

https://github.com/uwblueprint/richmond-centre-for-disability/assets/24593519/e81389cc-1acb-460f-9eff-7d0de649a992